### PR TITLE
Simplify the code

### PR DESCRIPTION
**Pull request by sitefinitysteve**
I have had a couple people pull down the hello world, and they get super intimidated by the object prototyping in the model to the point where they think {N} is more complex than it is.

I would like to suggest these changes as a way to simplify the learning curve for new users.

### CODE REVIEW

File: main-view-model.js [hunk 1]
```diff
@@ -1,21 +1,9 @@
...
-        else {
-            this.set("message", this.counter + " taps left");
-        }
-    };
-    return HelloWorldModel;
-})(observable.Observable);
-exports.HelloWorldModel = HelloWorldModel;
-exports.mainViewModel = new HelloWorldModel();
+var viewModel;
+var observable = require('data/observable').Observable;
```
Comment (tjvantoll) [inline main-view-model.js]: This should be upper case. `var Observable`

**Comment by tjvantoll** [comment]
I’m in favor of this change 👍 The TypeScript generated code is definitely intimidating.

**Comment by jlooper** [comment]
I'd chime in that the hello world example could not only be simpler code, it could be more functional as an example. I think Ionic's example app might be a good sample to emulate.

**Comment by sitefinitysteve** [comment]
UNSOLICTED! https://plus.google.com/u/0/116589386210724306149/posts/UHATyXmEwm2?cfem=1

**Comment by hshristov** [comment]
:-1: 

Exporting instances leads to memory leak
```
+module.exports = viewModel; 
```

Once `main-view-model` is required `viewModel` instance will stay alive even after page is no longer needed. So instantiating it when set to `bindingContext` is much cleaner and safer.

I know that the current template have the same issue (e.g. memory leak) but we will fix it.

**Comment by vakrilov** [comment]
Closing this PR in favor of #41

**Comment by sitefinitysteve** [comment]
@hshristov I was not aware that causes leaks, thanks for the heads up, I have some refactoring to do

**Pull request closed without merging**